tests/model/test_model.py

```diff
@@ -785,8 +785,7 @@ def test__handle_message_event(self, mocker, user_profile, response,
 
     @pytest.mark.parametrize('found_newest', [
         (True),
-        pytest.param(False,
-                     marks=pytest.mark.xfail(reason="Unread count bug")),
+        (False),
```

Comment:
I understand this is a simple adjustment from the previous commit and so a small diff, but I wonder if this should shrink onto one line now.